### DISCUSSION TITLE
Supporting Android Auto

### DISCUSSION THREAD

**Discussion by natema** [gokadzev/Musify#411] [Ideas]
It would be great if the app would work with Android Auto. 
I saw that this was proposed in #103, but I didn't find any later discussion on this feature here.

**Comment by gokadzev** [comment]
> It would be great if the app would work with Android Auto. I saw that this was proposed in #103, but I didn't find any later discussion on this feature here.

@natema  I can give it a try, could you test it?

**Comment by natema** [comment]
Sure!

**Reply by gokadzev** [reply]
> Sure!

@natema Perfect! here is the build: https://github.com/gokadzev/Musify/releases/download/master/Musify.apk


(You may need to back up data, uninstall the existing version and install this one)

**Reply by gokadzev** [reply]
let me know if it works after testing...

**Reply by natema** [reply]
Of course, I'll let you know tomorrow.

**Reply by natema** [reply]
Unfortunately Musify is not showing up in Android Auto.  
Let me know if there's any helpful info I could provide.

**Reply by gokadzev** [reply]
> Unfortunately Musify is not showing up in Android Auto. Let me know if there's any helpful info I could provide.

Maybe this is the case? https://www.reddit.com/r/AndroidAuto/comments/10k0uyc/missing_apps_in_android_auto_launcher_menu/

From the app side there should be no problem as I believe I have added everything that is needed to make it compatible with Android Auto.

**Reply by natema** [reply]
> Maybe this is the case? https://www.reddit.com/r/AndroidAuto/comments/10k0uyc/missing_apps_in_android_auto_launcher_menu/

Thanks, I will test tomorrow!

**Reply by natema** [reply]
Unfortunately I can't find the settings suggested in the Reddit post :disappointed: .
Perhaps they're not available anymore?
I'm using Android Auto 12.5.643034-release (125643034).

**Reply by natema** [reply]
Never mind, I found out how to activate developer mode. I'll let you know.

**Reply by natema** [reply]
@gokadzev it works! 🎉

**Reply by natema** [reply]
Just a small feedback on the current interface (let me know if it's more appropriate to move this to a separate discussion): when playing a playlist, the "forward" and "backward" buttons of the car skip to the next or previous song, as I'd expect, but when I select just one song, they just skip a bit forward and back in the same track. I'd find it more practical to be able to skip to the next suggested song, as in the YouTube Music app.
